components/commons/blog/postPreviewItem/postPreviewItem.global.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div class="bg-light-surface dark:bg-dark-surface flex flex-col-reverse lg:flex-row mb-8 rounded py-4 px-6 hover:bg-gray-200 transition-colors duration-300 ease-linear">
+    <div class="w-full lg:w-3/4 flex flex-col justify-between pr-4">
+      <div class="">
+        <div class="font-semibold text-xl mb-3">
+          <nuxt-link :to="`/blog/${post.slug}`" tag="a">{{ post.title }}</nuxt-link>
```

Comment:
We should semantically emphasize title with heading tag of some level. I suggest 

```
<h{n}>
  <nuxt-link/>
</h{n}>
```
You can use h1 if you wrap card in `article` tag, otherwise it should be more secondary heading level.

Reply:
Also, may be color it with primary green on hover?